src/memary/agent/base_agent.py

```diff
@@ -58,11 +56,14 @@ def __init__(
         system_persona_txt,
         user_persona_txt,
         past_chat_json,
+        llm_model_name,
```

Comment:
If you want to set `llama3` as the llm_model_name, what about writing it as default?
`llm_model_name="llama3"`